src/segment-loader.js

```diff
@@ -1396,8 +1396,11 @@ export default class SegmentLoader extends videojs.EventTarget {
     // to the initSegment cache
     if (simpleSegment.map) {
       simpleSegment.map = this.initSegmentForMap(simpleSegment.map, true);
-      // move the map bytes onto the segment loader's segment state object
+      // move over media segment request init segment additions
       segmentInfo.segment.map.bytes = simpleSegment.map.bytes;
+      segmentInfo.segment.map.tracks = Object.assign({}, simpleSegment.map.tracks);
```

Comment:
seems like increasing playbackRate showed a race condition where an alt audio track wouldn't have, tracks or timescales set correctly on a subsequent request.

Reply:
Maybe this should be in the commit message as well? Or even split the test changes from this?

Reply:
I will split it out